act1.sh

```diff
@@ -1,13 +1,14 @@
 # 1. In this directory, create a new directory called star_wars. Example answer: mkdir star_wars
 mkdir star_wars
 # 2. In the star_wars folder, create two new directories: empire and rebellion (This can be done in two commands, but how would you do it in one?)
-
+mkdir empire rebellion 
 # 3. Inside the empire directory, create a file called darth_vader.txt 
-
+touch darth_vader.txt
 # 4. Use the force (or your echo) to add the text "...heavy breathing..." to the darth_vader.txt file (Don’t remember how to do this? Use the ‘other force’, known as Google!)
 
 # 5. Inside the empire directory, create a file called emperor_palpatine.txt
-
+touch emperor_palpatine.txt
 # 6. Inside the empire directory, create a directory called death_star
-
+touch death_star
 # 7. Move darth_vader.txt into the death_star
+mv ~/death_star/pjs/favorite.darth_vader ~death_start/darth_vader/
```

Comment:
I'm unsure what your absolute path would look like and it seems that you were trying to move a file called `favorite.dath_vader` into a directory called `darth_vader`.  Be sure to thoroughly read the prompt.  It states to move `darth_vader.txt` into the `death_star` directory.